VRCFaceTracking/OSC/OSCParams.cs

```diff
@@ -9,12 +9,13 @@ namespace VRCFaceTracking.OSC
     public class OSCParams
     {
         private const string DefaultPrefix = "/avatar/parameters/";
+        private const string CurrentVersionPrefix = "v2/";
```

Comment:
Kinda makes me think we should include some sort of versioning within the IParameter side of the parameter eventually, and use OSCParams or another wrapper that enforces it (like with the base parameters getting added to AllParameters for eg.). 

I think for now this works pretty good and we can always revisit it ofc!